docs/examples/howto/howto.md

```diff
@@ -1,24 +1,10 @@
----
-orphan: true
----
-
-(howto-examples)=
-# How-to guides
-
-:::{attention}
-This section presents various workflows using the example executables.
-These executables are **deprecated**, and the python bindings should be used
-to run example workflows (see [](python-bindings)).
-:::
+# How-to guides for the Examples
```

Comment:
I think we are now not mentioning anywhere, that the examples are deprecated.

Reply:
The examples standalone executables are deprecated. Ultimately, I want to have updated guides like this for the python examples workflows.